pkg/status/status_server.go

```diff
@@ -236,6 +238,23 @@ func (s *Server) loggersHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) accesslogHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		log.Errorf("accesslogHandler export POST method but get %v", r.Method)
+		return
+	}
+
+	var info bool
+	accesslogInfo := r.URL.Query().Get("enable")
+	if accesslogInfo == "true" {
+		info = true
+	} else {
+		info = false
+	}
+	s.xdsClient.WorkloadController.MetricController.EnableAccesslog.SetAccesslogTrigger(info)
```

Comment:
The calling stack is too deep, why not provide a method to `MetricController` or even to `WorkloadController`